ros/api/v0/hosts.py

```diff
@@ -274,19 +274,20 @@ def get(self, host_id):
         query = query.limit(limit).offset(offset)
         query_results = query.all()
 
-        if query_results:
-            performance_history = []
-            for profile in query_results:
-                performance_record = profile.display_performance_score
-                performance_record['report_date'] = profile.report_date
-                performance_history.append(performance_record)
-
-            record = {'inventory_id': host_id}
-            record['performance_history'] = performance_history
-        else:
+        if not query_results:
             abort(404, message="System {} doesn't exist"
                   .format(host_id))
 
-        return build_paginated_system_list_response(
+        performance_history = []
+        for profile in query_results:
+            performance_record = profile.display_performance_score
+            performance_record['report_date'] = profile.report_date
+            performance_history.append(performance_record)
+
+        record = {'performance_history': performance_history}
```

Comment:
You could directly pass `performance_history` in data and no need to declare `record`, right?